src/main/webapp/WEB-INF/views/admin/project/list-js.jsp

```diff
@@ -710,13 +710,19 @@
 			var distributionStatus = $("#distributionStatus").val();
 			var completeFlag = $("#completeFlag").val();
 			var dropFlag = $("#dropFlag").val();
-			
-			if(reason.split(" ").join("") == ""){
-				alertify.alert("Please leave a comment.");
-				$("#reason").next(".retxt").show();
-				
-				return false;
-			}
+            var commentFlag = true;
```

Comment:
It looks like it needs indentation.

Reply:
I think so too!
Sorry for not checking this part.